flatten-me.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+"""Flatten a list of strings and lists into a new list of discrete elements."""
+
+
+def flatten_me(lst):
+    """Parse elements in a list and append a new list to return."""
+    idx = 0
+    new_list = []
+    while idx < len(lst):
```

Comment:
Generally, for something like this, it's better to just iterate over values in the list i.e. `for item in lst:`